backend/Lua/LuaLocalReference.cc

```diff
@@ -43,6 +43,29 @@ void ensureNonnull(int index) {
   throw Exception("NullPointerException");
 }
 
+bool judgeIsArray(int index)
+{
+  auto lua = currentLua();
+  int currectArrIndex = 0;
+  bool isArray = true;
+
+  lua_pushnil(lua); 
+  
+  while (lua_next(lua, index))
```

Comment:
In Lua you can't really check it's an Array or Object, and it's also documented clearly.
I don't think this check is good.